platform/src/main/java/org/stellar/anchor/platform/service/SimpleInteractiveUrlConstructor.java

```diff
@@ -70,12 +75,15 @@ String constructToken(Sep24Transaction txn, Map<String, String> request) {
         (isEmpty(txn.getSep10AccountMemo()))
             ? txn.getSep10Account()
             : txn.getSep10Account() + ":" + txn.getSep10AccountMemo();
+    ClientsConfig.ClientConfig clientConfig =
```

Comment:
For custodial wallets you should also get name by account (transaction may not have a clientDomain)
If clientDomain is missing, should validate that client type is custodial

Reply:
By account, do you mean the signing key in the client config?

Reply:
Yes

Reply:
Done